app.js

```diff
@@ -0,0 +1,7 @@
+import config from './config/config.json';
+import { User, Product } from './models';
+
+console.log(config.name);
+
+let user = new User();
```

Comment:
consider use `const` everywhere where variable is not mutating. If there is mutation later - use `let`